CONTRIBUTING.md

```diff
@@ -24,29 +26,73 @@ below will make our lives easier, and by extension make us more likely to
 include your changes.
 
 * Commits should focus on one thing at a time. Do include whatever you need to
-  make your change work, but avoid putting unrelated changes in the same commit.
-  Preferably, one change in functionality should be in exactly one commit.
+  make your change work, but avoid putting unrelated changes in the same
+  commit. Preferably, one change in functionality should be in exactly one
+  commit—and commits should be atomic (i.e. try not to commit broken or
+  intermediate states of the code).
 * pep8ify your code before you commit. We don't worry about line length much
   (though it's good if you do keep lines short), but you should try to follow
   the rest of the rules.
 * Test your code before you commit. We don't have a formal testing plan in
-  place, but you should make sure your code works as promised before you commit.
+  place, but you should make sure your code works as promised before you
+  commit.
 * If you have cloned the repository to your personal computer, you can use our
   provided git hooks to automatically check whether your code style is up to
   Sopel's requirements and that tests still pass, before committing your code.
   The git hooks can be enabled by running `make install-hooks`, and disabled by
   running `make uninstall-hooks`,  in your root `sopel` folder.
 * Make your commit messages clear and explicative. Our convention is to place
-  the name of the thing you're changing in at the beginning of the
-  message, followed by a colon: the module name for modules, docs for documentation files,
-  coretasks for coretasks.py, db for the database feature, and so on.
+  the name of the thing you're changing in at the beginning of the message,
+  followed by a colon: the module name for modules, docs for documentation
+  files, coretasks for coretasks.py, db for the database feature, and so on.
```

Comment:
```suggestion
  the name of the thing you're changing in at the beginning of the message,
  followed by a colon: the plugin name for plugins, "docs" for documentation
  files, "coretasks" for `coretasks.py`, "db" for the database feature, etc.
```

Tweaks inspired by @Exirel's suggestion on line 47, but I had to integrate them to avoid a conflict when applying.